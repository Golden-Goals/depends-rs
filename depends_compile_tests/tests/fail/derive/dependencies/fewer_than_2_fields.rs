use depends::derives::Dependencies;

#[derive(Dependencies)]
struct Foo {
    bar: usize,
}

#[derive(Dependencies)]
struct Bar {}

fn main() {}
