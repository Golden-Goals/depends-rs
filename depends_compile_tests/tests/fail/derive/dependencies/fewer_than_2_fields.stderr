error: Dependencies must have at least 2 fields. Use `depends::Dependency` for a single dependency.
 --> tests/fail/derive/dependencies/fewer_than_2_fields.rs:3:10
  |
3 | #[derive(Dependencies)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Dependencies` (in Nightly builds, run with -Z macro-backtrace for more info)

error: Dependencies must have at least 2 fields. Use `depends::Dependency` for a single dependency.
 --> tests/fail/derive/dependencies/fewer_than_2_fields.rs:8:10
  |
8 | #[derive(Dependencies)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Dependencies` (in Nightly builds, run with -Z macro-backtrace for more info)
